src/components/LazyLog/index.tsx

```diff
@@ -347,6 +349,7 @@ export default class LazyLog extends Component<LazyLogProps, LazyLogState> {
         },
         caseInsensitive: false,
         enableGutters: false,
+        wrapLines: true,
```

Comment:
Move this down after the last `enable` so its not mixed in the middle and more alpha order?